### DISCUSSION TITLE
How to unpublish?

### DISCUSSION THREAD

**Discussion by lyra-white** [webstudio-is/webstudio-community#99] [General]
By default each project has a subdomain on wstd.io. Is there an easy way to unpublish the site?

https://doublequotesbug-bkgzs.wstd.io/

I created this project. After deleting it, it is still available on the wstd.io subdomain.

Wix has a simple functionality: https://support.wix.com/en/article/wix-editor-unpublishing-your-site

**Comment by kof** [comment]
We need to implement that https://github.com/webstudio-is/webstudio/issues/2891

Current workaround is to just clear the content and publish again.
